README.adoc

```diff
@@ -157,6 +157,11 @@ The https://github.com/input-output-hk/plutus-starter[plutus-starter] project pr
 This section contains information about how to build the project's artifacts for independent usage.
 For development work see <<how-to-develop>> for more information.
 
+[NOTE]
+====
+You may also want to consult the https://plutus-community.readthedocs.io/en/latest/[Plutus community documentation] for specific details regarding your particular environment, especially if you are a MacOS user.
```

Comment:
I think it would nice to explicitly state that it's especially useful for M1/M2 users.